README.md

```diff
@@ -0,0 +1,8 @@
+# package-dropbear-static
+
+- dropbear is a SSH 2 server and client designed to be small enough to be used in small memory environments, while still being functional and secure enough for general use.
+
+- This package was a customised package which extracted static binaries of `dropbear` from `.deb` packages in the release, and was customised for Garden Linux
+
```

Comment:
please add a sentence on what we used it for (ask @yeoldegrove for a bit more context)